metrictank.go

```diff
@@ -304,7 +304,7 @@ func main() {
 	/***********************************
 	    Start the ClusterManager
 	***********************************/
-	cluster.Start()
+	cluster.Manager.Start()
```

Comment:
why this change?

Reply:
That's not necessary anymore, it was necessary in an earlier version of this PR, removing this change.